src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -939,6 +939,13 @@ public IPointerTypeSymbol CreatePointerTypeSymbol(ITypeSymbol pointedAtType)
 
         protected abstract IPointerTypeSymbol CommonCreatePointerTypeSymbol(ITypeSymbol elementType);
 
+        public IFunctionPointerTypeSymbol CreateFunctionPointerTypeSymbol(ITypeSymbol returnType, RefKind returnRefKind, ImmutableArray<ITypeSymbol> parameterTypes, ImmutableArray<RefKind> parameterRefKinds)
```

Comment:
>CreateFunctionPointerTypeSymbol [](start = 42, length = 31)

Please add a doc comment, including for parameters, what values are allowed, etc.